### DISCUSSION TITLE
Add m-0-auto class to achieve margin: 0 auto;

### DISCUSSION THREAD

**Discussion by carlosmori** [tailwindlabs/tailwindcss#3480] [Ideas]
Hey guys, 
do you think is a good idea to add a class like m-0-auto to achieve margin: 0 auto.

If you think is a good idea I would like to make a PR it would be my first collab!

Thanks and amazing library!

**Comment by trm217** [comment]
You can achieve this by just using .mx-auto

**Reply by carlosmori** [reply]
Got it, thanks!
